### ISSUE TITLE
💪 Leaderboards

### ISSUE THREAD

**Issue by luttje** [luttje/gmod-experiment-redux#73]
> _(copied from Discord for now, will expand at a later time:)_
> Persistent progress, despite resetting. We hope to make Experiment a unique game where a single 'match' lasts a month, we call such a match 'An Epoch'. The race to wealth, power and the top of the leaderboards will make for an exciting Epoch each month.

**Comment by Nelaif** [comment]
I agree with the monthly resets. Being super wealthy is a little sketchy. 
In terms of character development, maybe traits can be unlocked over time when a character survives the epoch and was actively contributing to the livelyhood of the server. Maybe some rewards or cosmetics, maybe extra perk / skill slots.

**Comment by luttje** [comment]
There must be absolutely no advantage to gameplay in the rewards we give, everything must be cosmetic only. I want completely new players that have natural talent to be on equal grounds with players that have been playing for a few months. So only glory and cosmetic items.

**Comment by luttje** [comment]
I've updated the main issue a bit with some reasoning and background on the leaderboard system

**Comment by luttje** [comment]
First version of the leaderboards is now live. They can be viewed online only atm: https://leaderboards.experiment.games/

They sync every night around midnight (UTC)
